backend/src/core/common/port/cloudinary.ts

```diff
@@ -0,0 +1,6 @@
+import { UploadApiResponse } from 'cloudinary';
+import { CloudinaryUploadDto } from '~/shared/types/types';
+
+export interface CloudinaryPort {
```

Comment:
You shouldn't use a cloud name, it's better to abstract away the names of the services that we use in our business logic. It should be something like `ImageStoragePort`